commit f6cd796e0b63460da231cdb3269e8fca5b279286 (refs/changes/60/5160/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-11-28T15:54:56-08:00 (10 months ago)
    
    TRITON-990 'sdcadm check-health' should stop writing out a trace log

diff --git a/lib/cli/do_check_config.js b/lib/cli/do_check_config.js
index 3e8c218..51aa311 100644
--- a/lib/cli/do_check_config.js
+++ b/lib/cli/do_check_config.js
@@ -491,7 +491,7 @@ do_check_config.help = (
     '\n' +
     '{{options}}'
 );
-do_check_config.logToFile = false;
+
 
 // --- exports
 
diff --git a/lib/cli/do_check_health.js b/lib/cli/do_check_health.js
index 41c3dfe..0fe5b58 100644
--- a/lib/cli/do_check_health.js
+++ b/lib/cli/do_check_health.js
@@ -305,9 +305,10 @@ do_check_health.help = (
     '    cnapi cn-agent\n'
 
 );
-do_check_health.logToFile = false;
+
 
 // --- exports
+
 module.exports = {
     do_check_health: do_check_health
 };
